src/Core/Domain/EnvironmentSettings.ts

```diff
@@ -71,8 +71,8 @@ interface WithHandlers<ValueType extends Value> {
   initial: ValueType;
   value: ValueType;
   set: (value: ValueType) => void;
-  update: (value: ValueType) => Promise<Maybe.Maybe<string>>;
-  reset: () => Promise<Maybe.Maybe<string>>;
+  update: (value: ValueType) => void;
```

Comment:
We were returning a string in certain cases, was that for error handling?

Reply:
Yes, now the error handling is at the top